README.md

```diff
@@ -1,138 +1,214 @@
+
+
+ Apollo 2.5 is loaded with new modules and features, but needs to be callibrated and configured perfectly before you take it for a spin. Please review the prerequisites and installation steps in detail to ensure that you are well equipped to build and launch Apollo. You could also check out Apollo's architecture overview for a greater understanding on Apollo's core technology and platform. 
+
+[Want to contribute to our code?](https://github.com/ApolloAuto/apollo/blob/master/CONTRIBUTING.md) follow this guide.
+
+## Prerequisites
+
+#### Basic Requirements:
+
+* Vehicle equipped with by-wire system, including but not limited to brake by-wire, steering by-wire, throttle by-wire and shift by-wire (Apollo is currently tested on Lincoln MKZ)
+
+* A machine with a 4-core processor and 6GB memory minimum
+
+* Ubuntu 14.0.4
+
+* Working knowledge of Docker
+
+~~~ 
+ - Please note, it is recommended that you install the versions of Apollo in the following order: 
+ 1.0 > 1.5 > 2.0 > 2.5
+ The reason behind this recommendation is that you need to confirm whether individual hardware components 
+ and modules are functioning correctly and clear various version test cases,
+ before progressing to a higher more capable version for your safety and the safety of those around you.
+
+ - Please note, if you do not have a vehicle, proceed to the Installation - without hardware 
+~~~
+
+#### Individual Version Requirements:
+
+The following diagram highlights the scope and features of each Apollo release:
+
+![image alt text](images/Apollo_versions.png)
+
+[**Apollo 1.0:**](https://github.com/ApolloAuto/apollo/blob/master/docs/quickstart/apollo_1_0_hardware_system_installation_guide.md) 
+
+Apollo 1.0 also referred to as the Automatic GPS Waypoint Following, works in an enclosed venue such as a test track or parking lot. This installation is necessary to ensure that Apollo works perfectly with your vehicle. The diagram below lists the various modules in Apollo 1.0.
+
+![image alt text](images/Apollo_1.png)
+
+**For Setup:**
+
+* **Hardware**:
+
+    * Industrial PC (IPC)
+
+    * Global Positioning System (GPS)
+
+    * Inertial Measurement Unit (IMU)
+
+    * Controller Area Network (CAN) card
+
+    * Hard drive
+
+    * GPS Antenna
+
+    * GPS Receiver
+
+* **Software**:
+
+    * Apollo Linux Kernel (based on Linux Kernel 4.4.32)
+
+[**Apollo 1.5:**](https://github.com/ApolloAuto/apollo/blob/master/docs/quickstart/apollo_1_5_hardware_system_installation_guide.md) 
+
+Apollo 1.5 is meant for fixed lane cruising. With the addition of LiDAR, vehicles with this version now have better perception of its surroundings and can better map its current position and plan its trajectory for safer maneuvering on its lane. Please note, the modules highlighted in Yellow are additions or upgrades for version 1.5.
+
+![image alt text](images/Apollo_1_5.png)	
+

```

Comment:
Please create the image directory for all your images inside the folder "docs" and refer your images in README from there. Try not to create a folder called "images" in Apollo root directory.